jest.config.js

```diff
@@ -159,15 +157,11 @@ module.exports = {
     'src/store/actions/bookmarks.js',
     'src/utils/api/transaction/lsk.js',
     'src/utils/transaction.js',
-    'src/store/actions/voting.js',
     'src/components/shared/filterDropdownButton/addressFilter.js',
     'src/components/shared/transactionsTable/transactionRow.js',
     'src/store/selectors.js',
     'src/components/shared/transactionSummary/index.js',
     'src/routes.js',
-    'src/store/actions/account.js',
-    'src/store/actions/blocks.js',
-    'src/store/actions/bookmarks.js',
```

Comment:
There are still duplicates of these files left in lines 153, 155 and 157. Please remove them and ensure that the files show up in coverage

Reply:
There are also non existent files in lines 59 - 64 which should be removed